LayoutTests/fast/hdr/hdr-basic-canvas-layer-dump-expected.txt

```diff
@@ -0,0 +1,56 @@
+
+(GraphicsLayer
```

Comment:
This test might be better under `compositing/hdr` since it dumps GraphicsLayers.

Reply:
This test was moved to `compositing/hdr/basic-hdr-canvas.html`